current_fall2025/scripts/gold_build_bpl_vector.py

```
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)